### ISSUE TITLE
Sliders in APCA Contrast Calculator are not keyboard accessible

### ISSUE THREAD

**Issue by kinseytamsin** [Myndex/SAPC-APCA#143]
### Does this relate to a web app or tool (i.e. you're an end user) OR is this about code or code integration (you're a developer).
I am an end user. This relates to the APCA Contrast Calculator web application at <https://apcacontrast.com/>.

### Describe the bug
The HSV sliders are not keyboard accessible. This issue seems to be inherited from upstream https://github.com/jaames/iro.js/issues/237, but unfortunately that project also seems to be [on hiatus since 2022](https://github.com/jaames/iro.js/issues/217#issuecomment-1214403290), so it doesn't seem likely to be addressed there anytime soon.

### To Reproduce
Two main ways:

#### Method 1
1. Navigate to https://apcacontrast.com/ in a web browser.
2. Press <kbd>Tab</kbd> until focus is on the <samp>Text / Icon Color</samp> text input box.
3. Press <kbd>Tab</kbd> one more time to move to the next sequentially focusable element.

**Expected behavior:** The focus moves to the handle of the first slider.

**Actual behavior:** The focus moves to the other text input box.

#### Method 2
1. Navigate to https://apcacontrast.com/ in a web browser.
2. Click on any of the slider handles.
3. Press the <kbd>Left Arrow</kbd> key or the <kbd>Right Arrow</kbd> key.

**Expected behavior:** The slider handle moves and the corresponding value changes.

**Actual behavior:** Nothing happens.

### System or device information (please complete the following):
- **Device:** PC
- **OS:** Windows 11
- **OS version:** 23H2 Build 22631
- **Browser:** Firefox
- **Browser version:** 145.0.1 Build 20251118022223

### Additional context
- [WCAG 2.2 Success Criterion 2.1.1, "Keyboard"](https://www.w3.org/TR/WCAG22/#keyboard)
- Not *directly* addressed by this issue, but also currently failed by this UI and a change that fixes this issue would create the perfect opportunity to address this as well: [WCAG 2.2 Success Criterion 4.1.2, "Name, Role, Value"](https://www.w3.org/TR/WCAG22/#name-role-value)

**Comment by Myndex** [comment]
Yes, we did not develop these, and I believe James was working on an upgrade to implement more advanced color models, we'll look into this. That said, the APCA calculator is a technology demonstrator, not a production tool.

**Comment by jaames** [comment]
Hi, I'm the developer of iro.js. I can confirm that unfortunately iro.js does not currently support keyboard controls and the project has been on hiatus for some time. The rewrite I was working on already has ARIA support and keyboard input, and while I also had to set that aside for a while, I'll put a higher priority on finishing/releasing it.
